backend/btrixcloud/users.py

```diff
@@ -184,6 +187,9 @@ async def get_user_info_with_orgs(self, user: User) -> UserOut:
                         if user.is_superuser
                         else org.users.get(str(user.id))
                     ),
+                    readOnly=org.readOnly,
+                    readOnlyReason=org.readOnlyReason,
+                    subscription=org.subscription,
```

Comment:
Should test to make sure this doesn't cause any issues when `info_out_cls` is set to `UserOrgInfoOut`

Reply:
Tested manually and also `/register` endpoint in tests, tests out this path, should be ok!